keyboards/ut472/keymaps/praxling_dvorak/keymap.c

```diff
@@ -0,0 +1,459 @@
+/* Copyright 2018 Carlos Filoteo
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define L3_TAB LT(3, KC_TAB)
+#define L5_
+#define KC_LOCK RGUI(KC_L)
+#define L4_SPACE LT(4, KC_SPC)
+#define KC_HYP LCTL(LSFT(LALT(KC_LGUI)))
+#define HYP5 LM(5, MOD_LCTL | MOD_LSFT | MOD_LALT | MOD_LGUI)
+#define TD_LAYER TD(TD_LAYER_SWAP)
+
+typedef struct {
+  bool is_press_action;
+  int state;
+} tap;
+
+enum custom_keycodes {
+    TOG_OS = SAFE_RANGE,
+    OSM_SHIFT,
+
+    // shortcuts
+    UC_COPY,
+    UC_PASTE,
+    UC_CUT,
+    UC_UNDO,
+    KC_COMMENT,
+
+    // media
+    KC_MUTING,
+    KC_VUP,
+    KC_VDOWN,
+
```

Comment:
`KC_MUTE`, `KC_VOLU`, and `KC_VOLD` should work in Windows, Linux and macOS environments. If you use those keycodes in your keymap, you don't need Lines 42-46, or the corresponding block in `process_record_user()` (Lines 254-280).